httpServer/ui/components/dashboard.js

```diff
@@ -340,6 +345,30 @@ export default class Dashboard extends React.Component {
         });
     }
 
+    toggleActiveState() {
+        this.saveSetting('ACTIVE', this.state.settings.ACTIVE === 0 ? 1 : 0);
+    }
+
+    closeButton_OnClick() {
```

Comment:
not consistent with the rest of the naming in this project. Also it's not really a closebutton, more of an on/off button. Maybe confirmActiveStateChange would be a better option?

Reply:
changed to confirmActiveStateChange